Client/Frontend/Browser/Browser.swift

```diff
@@ -117,6 +117,8 @@ class Browser: NSObject, BrowserWebViewDelegate {
             configuration!.userContentController = WKUserContentController()
             configuration!.preferences = WKPreferences()
             configuration!.preferences.javaScriptCanOpenWindowsAutomatically = false
+            configuration!.allowsInlineMediaPlayback = true
```

Comment:
Just to confirm, this will only allow webkit-playsinline videos to play inline correct?